src/Security/Security/Cmdlets/SqlVulnerabilityAssessment/Common/SqlVulnerabilityAssessmentSetBaselineBase.cs

```diff
@@ -0,0 +1,65 @@
+        [ValidateNotNullOrEmpty]
+        public string ResourceId { get; set; }
+
+        [Parameter(ParameterSetName = ParameterSetNames.ResourceIdWithBaselineObject, Mandatory = true, HelpMessage = ParameterHelpMessages.WorkspaceId)]
+        [Parameter(ParameterSetName = ParameterSetNames.OnPremMachinesWithBaselineObject, Mandatory = true, HelpMessage = ParameterHelpMessages.WorkspaceId)]
+        [Parameter(ParameterSetName = ParameterSetNames.InputObjectBaselineWithResourceId, Mandatory = true, HelpMessage = ParameterHelpMessages.WorkspaceId)]
+        [Parameter(ParameterSetName = ParameterSetNames.InputObjectBaselineWithOnPrem, Mandatory = true, HelpMessage = ParameterHelpMessages.WorkspaceId)]
+        [ValidateNotNullOrEmpty]
+        public string WorkspaceId { get; set; }
+
+        [Parameter(ParameterSetName = ParameterSetNames.ResourceIdWithBaselineObject, Mandatory = true, HelpMessage = ParameterHelpMessages.Server)]
+        [Parameter(ParameterSetName = ParameterSetNames.OnPremMachinesWithBaselineObject, Mandatory = true, HelpMessage = ParameterHelpMessages.Server)]
+        [Parameter(ParameterSetName = ParameterSetNames.InputObjectBaselineWithResourceId, Mandatory = true, HelpMessage = ParameterHelpMessages.Server)]
+        [Parameter(ParameterSetName = ParameterSetNames.InputObjectBaselineWithOnPrem, Mandatory = true, HelpMessage = ParameterHelpMessages.Server)]
+        [ValidateNotNullOrEmpty]
+        public string Server { get; set; }
+
+        [Parameter(ParameterSetName = ParameterSetNames.ResourceIdWithBaselineObject, Mandatory = true, HelpMessage = ParameterHelpMessages.Database)]
+        [Parameter(ParameterSetName = ParameterSetNames.OnPremMachinesWithBaselineObject, Mandatory = true, HelpMessage = ParameterHelpMessages.Database)]
+        [Parameter(ParameterSetName = ParameterSetNames.InputObjectBaselineWithResourceId, Mandatory = true, HelpMessage = ParameterHelpMessages.Database)]
+        [Parameter(ParameterSetName = ParameterSetNames.InputObjectBaselineWithOnPrem, Mandatory = true, HelpMessage = ParameterHelpMessages.Database)]
+        [ValidateNotNullOrEmpty]
+        public string Database { get; set; }
+
+        [Parameter(ParameterSetName = ParameterSetNames.OnPremMachinesWithBaselineObject, Mandatory = true, HelpMessage = ParameterHelpMessages.ComputerName)]
+        [Parameter(ParameterSetName = ParameterSetNames.InputObjectBaselineWithOnPrem, Mandatory = true, HelpMessage = ParameterHelpMessages.ComputerName)]
+        [ValidateNotNullOrEmpty]
+        public string ComputerName { get; set; }
+
+        [Parameter(ParameterSetName = ParameterSetNames.OnPremMachinesWithBaselineObject, Mandatory = true, HelpMessage = ParameterHelpMessages.VMUUID)]
+        [Parameter(ParameterSetName = ParameterSetNames.InputObjectBaselineWithOnPrem, Mandatory = true, HelpMessage = ParameterHelpMessages.VMUUID)]
+        [ValidateNotNullOrEmpty]
+        public string VMUUID { get; set; }
```

Comment:
Can the name of 'VMUUID' follow Pascal Case?

Reply:
I didn't know the right way to case thing since it's a shortcut for "Virtual Machine Universal unique identifier", i'll change it to Vmuuid, if you have another preferable casing powershell wise then please let me know.

Reply:
I prefer VmUuid or VMUuid, what do you think?

Reply:
going with VmUuid